app/assets/javascripts/browsered/worldpay-3ds-flex-ddc.js

```diff
@@ -54,7 +59,12 @@ const submitWithWorldpay3dsFlexDdcResult = form => {
       const { MessageType, SessionId, Status } = JSON.parse(event.data)
       if (MessageType === 'profile.completed') {
         if (Status === true && typeof SessionId === 'string') {
+          updateDdcStatusField('valid DDC result')
           addWorldpaySessionIdToForm(form, SessionId)
+        } else if (!Status) {
+          updateDdcStatusField('DDC result had Status of false')
```

Comment:
“DDC result did not have Status of true”? That would cover it not being there at all or not being a boolean.

Reply:
that's the equivalent of being false in javascript haha. I was gonna say 'DDC result has a Status that was not "true"' but i thought that made less sense haha